types/react-autosuggest/index.d.ts

```diff
@@ -1,76 +1,97 @@
-// Type definitions for react-autosuggest 8.0
+// Type definitions for react-autosuggest 9.3
 // Project: http://react-autosuggest.js.org/
-// Definitions by: Nicolas Schmitt <https://github.com/nicolas-schmitt>, Philip Ottesen <https://github.com/pjo256>, Robert Essig <https://github.com/robessog>, Terry Bayne <https://github.com/tbayne>
+// Definitions by: Nicolas Schmitt <https://github.com/nicolas-schmitt>
+//                 Philip Ottesen <https://github.com/pjo256>
+//                 Robert Essig <https://github.com/robessog>
+//                 Terry Bayne <https://github.com/tbayne>
+//                 Christopher Deutsch <https://github.com/cdeutsch>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
 import * as React from 'react';
-declare class Autosuggest extends React.Component<any> {}
+declare class Autosuggest extends React.Component<Autosuggest.AutosuggestProps> {}
```

Comment:
When this was `any` it wasn't actually doing any type checking when you used `<Autosuggest />`